### PR TITLE
New machines marked as NOT_WORKING

### PR DESCRIPTION
**Pull request by clawgrip**
----------------------------------
Games Family [Victor Fernandez (City Game), ClawGrip]

### CODE REVIEW

File: src/mame/drivers/gfamily.cpp [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+ -MAME v0.106 (May 16 2006)
+ -Nebula 2.1.5
+ -FB Alpha v0.2.94.98
+ -ZiNc 1.1
+
+
+PC motherboard plus an additional PCB for JAMMA, inputs and basic config.
+
+PC board:
+CPU: Intel Celeron 1,7GHz / 128kb / 400MHz SL6SC
```
Comment (cuavas) [inline src/mame/drivers/gfamily.cpp:18]: Please use English number format.

File: src/mame/drivers/gfamily.cpp [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+RAM: 256MB-DDR400
+PCB: 04/082006-SiS-651-6A6IXRMAC-00, Realtec ALC655 audio, Realtec RTL8201BL Ethernet (25.000 MHz xtal), Winbond W83194BG-648 (14.31818 xtal)
```
Comment (cuavas) [inline src/mame/drivers/gfamily.cpp:20]: One of the frequencies here doesn’t have a unit.

File: src/mame/drivers/gfamily.cpp [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+}
+
+void gfamily_state::machine_reset()
+{
+}
+
+void gfamily_state::gfamily(machine_config &config)
+{
+	// Basic machine hardware
+	PENTIUM4(config, m_maincpu, 1700000000); // Actually an Intel Celeron SL6SC 1,7GHz
```
Comment (cuavas) [inline src/mame/drivers/gfamily.cpp:84]: Please use digit grouping in large numbers to it’s easier to see the order of magnitude.

**Review by cuavas — CHANGES_REQUESTED** [review]

File: src/mame/drivers/gfamily.cpp (hunk 1 above)
Comment (clawgrip) [reply src/mame/drivers/gfamily.cpp]: Done, thanks!

File: src/mame/drivers/gfamily.cpp (hunk 2 above)
Comment (clawgrip) [reply src/mame/drivers/gfamily.cpp]: Done, thanks!

**Pull request merged**